tests/installation/scc_registration.pm

```diff
@@ -31,10 +31,9 @@ sub run {
 
 sub post_fail_hook {
     my ($self) = @_;
+    verify_scc;
+    investigate_log_empty_license;
```

Comment:
Minor point, can we flip it, so parent postfail hook is executed first? As we recover network, fix RO filesystem there.

Reply:
Flipped does not work: http://rivera-workstation.suse.cz/tests/2094#step/yast2_scc/163http://rivera-workstation.suse.cz/tests/2094#step/yast2_scc/163http://rivera-workstation.suse.cz/tests/2094#step/yast2_scc/163 I couldn't figure out why as there is a chain of post_failed_hooks plus multiple inheritance in some case. I'd prefer to put it there for the moment.

Reply:
Upload of the logs somehow fails, but I can also see that your branch is not up to date. Could you please pull latest changes and retry, if that fails again, we can leave it.

Reply:
Wait, it works, but only for this module as you wisely mentioned: http://rivera-workstation.suse.cz/tests/2096#step/scc_registration/258

Reply:
So, are you going to introduce more changes or we leave it for later?

Reply:
For the other two modules flipping it does not work after rebasing, i.e.: http://rivera-workstation.suse.cz/tests/2097#step/yast2_scc/500 (I remember I noticed during development, something to investigate for next task definitely)

Reply:
I flipped for `scc_registration` that works, the other two better for later.